src/index.html

```diff
@@ -11,8 +14,64 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="page__body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image-wrap">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="card-image"
+        />
+      </div>
+
+      <div class="card__info">
+        <h1 class="card__name">
```

Comment:
This tag use for title of the page, so you need to use the another tag here
```suggestion
        <h2 class="card__name">
```